README.md

```diff
@@ -106,6 +103,10 @@ Note that when using `docker-compose run`, the docker network will not be expose
 
 The `db` container is exposed on port `5433` of your host computer to make it easier to run commands on. For instance, you can open a `psql` session to it by running `psql -h localhost -p 5433 -d federalist -U postgres`.
 
+The admin client is running on port `3000` of hour host computer.
+
+Some aspects of the system aren't expected to be fully functional in a development environment. For example: the "View site", "View logs", and "Uploaded files" links associated with sites in the seed data do not reach working URLs.
```

Comment:
hmmm, `View Logs` should generally work...

Reply:
I think I grouped those three things together overly broadly in the notes I was working from here. "View logs" *is* a working URL — it 200s — but reports "Not found" and "This build does not have any build logs" for the links I've tried.

Reply:
That just means that no logs were seeded for those builds.

Reply:
I have removed my assertion about "View logs".